src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp

```diff
@@ -1,4 +1,4 @@
-/*****************************************************************************
+﻿/*****************************************************************************
```

Comment:
Looks like your text editor inserted a BOM. Our files use UTF-8 without a BOM.

Reply:
Ah I see, I'll fix that